liquibase/scripts/generate_liquibase_snapshots.sh

```diff
@@ -0,0 +1,94 @@
+    new_liquibase_file="liquibase-update-to-latest-${new_openmrs_version}.x.xml"
+    cp "api/src/main/resources/liquibase-update-to-latest-template.xml" "api/src/main/resources/org/openmrs/liquibase/updates/$new_liquibase_file"
+    
+    database_updater_class="api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java"
+    variable_name="CHANGE_SET_COUNT_FOR_GREATER_THAN_2_1_X"
+    
+    # Increase the variable value by 1
+    sed -i "s/\($variable_name *= *\)[0-9]\+/\1$(( $(grep -o "$variable_name *= *[0-9]\+" "$database_updater_class" | awk '{print $NF}') + 1 ))/" "$database_updater_class"
+    
+    insert_line="	<include file=\"org/openmrs/liquibase/updates/$new_liquibase_file\"/>"
+    sed -i -e '$!N;$!N;$i\'"$insert_line" "api/src/main/resources/liquibase-update-to-latest-from-1.9.x.xml"
+
+}
+
+function update_log_files_with_new_snapshots() {
+    sed -i 's/\(<include file="org\/openmrs\/liquibase\/snapshots\/schema-only\/liquibase-schema-only-\)\([^"]*\)\.x\.xml"\/>/\1'"$openMRS_version"'\.x.xml"\/>/' "api/src/main/resources/liquibase-schema-only.xml"
+    sed -i 's/\(<include file="org\/openmrs\/liquibase\/snapshots\/core-data\/liquibase-core-data-\)\([^"]*\)\.x\.xml"\/>/\1'"$openMRS_version"'\.x.xml"\/>/' "api/src/main/resources/liquibase-core-data.xml"
+    sed -i 's/\(<include file="org\/openmrs\/liquibase\/updates\/liquibase-update-to-latest-\)\([^"]*\)\.x\.xml"\/>/\1'"${new_openmrs_version}"'\.x.xml"\/>/' "api/src/main/resources/liquibase-update-to-latest.xml"
+}
+
+function update_snapshot_tests() {
+    change_log_java_file="api/src/main/java/org/openmrs/liquibase/ChangeLogVersions.java"
+    
+    # Use grep to check if the version already exists in the lists
+    if ! grep -q "${openMRS_version}" "$change_log_java_file"; then
+        # Add the new version to the lists in the Java class
+        sed -i '/private static final List<String> SNAPSHOT_VERSIONS = Arrays.asList(/s/);/,\ "'"${openMRS_version}"'"\);/' "$change_log_java_file"
+    fi
+    
+    if ! grep -q "${new_openmrs_version}" "$change_log_java_file"; then
+        sed -i '/private static final List<String> UPDATE_VERSIONS = Arrays.asList(/s/);/,\ "'"${new_openmrs_version}"'"\);/' "$change_log_java_file"
+    fi
+    
+    # Update the version in openmrs_version.txt only if it's a new version
+    if [ "${openMRS_version}" != "${new_openmrs_version}" ]; then
+        sed -i "s/\(openmrs_version.txt=\).*/\1${new_openmrs_version}.0-SNAPSHOT/" "liquibase/scripts/fix_liquibase_snapshots.sh"
+    fi
+}
+function test_liquibase_snapshots() {
+    cd liquibase/
+    . scripts/test_liquibase_snapshots.sh "${1}" "${2}"
+    cd ..
+}
+function echo_usage() {
+	echo "usage: . liquibase/scripts/create_liquibase_snapshots.sh <username> <password>"
+}
+
+project_version=$(grep -m 1 '<version>' pom.xml | sed -n 's/.*<version>\(.*\)<\/version>.*/\1/p') 
+openMRS_version=$(echo $project_version | sed "s/\([0-9]*\.[0-9]*\)\..*/\1/")
```

Comment:
Is there a reason why for some you are using capital letters and then others lower case? `openMRS_version` vs `new_openmrs_version`

Reply:
No reason to name like this